client/client.go

```diff
@@ -134,9 +138,9 @@ func getTokenFromEnv() string {
 	return ""
 }
 
-func getSpacesTokenFromEnv() (string, string) {
-	spacesAccessKey := os.Getenv("SPACES_ACCESS_KEY_ID")
-	spacesSecretKey := os.Getenv("SPACES_SECRET_ACCESS_KEY")
+func getSpacesTokenFromEnv() (spacesAccessKey string, spacesSecretKey string) {
```

Comment:
`confusing-results: unnamed results of the same type may be confusing, consider using named results`

Reply:
I don't think we need named results in this instance.

```suggestion
func getSpacesTokenFromEnv() (string, string) {
```